test/functionalTest/cases/4285_timeout_field_in_subscriptions_for_mqtt_and_mqttCustom_cases/timeout_field_in_subscriptions_for_mqttCustom.test

```diff
@@ -0,0 +1,603 @@
+# Copyright 2023 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Is this .test based in some existing test for HTTP timeout? In that case, could you provide name/link to that file, pls?

Reply:
`test/functionalTest/cases/3842_per_sub_httpTimeout/per_sub_httpTimeout_crud_custom.test`